src/data/splash-messages.ts

```diff
@@ -176,41 +179,146 @@ const commonSplashMessages = [
   "timeForYourDeliDelivery",
   "goodFirstImpression",
   "iPreferRarerCandies",
+  "pocketRoguelite",
+  "porygonDidNothingWrong",
+  "critMattered",
+  "pickupNotRequired",
+  "stayHydrated",
+  "alsoTryCobblemon",
+  "alsoTryPokeDoku",
+  "mySleepStyleIsDoesnt",
+  "makeYourOwnWorldChampDifference",
+  "yoChampInTheMaking",
+  "notLiableForDecisionAnxiety",
+  "theAirIsTastyHere",
+  "continue",
+  "startANewRunToday",
+  "neverGiveUp",
+  "theresAlwaysNextTime",
+  "oneTwoThreeAndPoof",
+  "yourPokemonOnlyGoToLevelOneHundred",
+  "theBattlesWillBeLegendary",
+  "levelCurveBetterThanJohto",
+  "alsoTryShowering",
+  "wellStillBeHere",
+  "weHopeToSeeYouAgain",
+  "aHealthyTeamCanMeanGreaterRewards",
+  "aWildPokemonAppeared",
+  "isThisThingOn",
+  "needsMoreTesting",
+  "whoChecksStatChanges",
+  "whenTwoTrainerseyesMeet",
```

Comment:
```suggestion
  "whenTwoTrainersEyesMeet",
```

Spelled differently in the locales Repo